### ISSUE TITLE
[iOS] Cannot download predictive dictionaries

### ISSUE THREAD

**Issue by jahorton** [keymanapp/keyman#1912]
**Describe the bug**
Attempting to download a predictive dictionary on my phone fails to visibly respond in any capacity.

**To Reproduce**
1. Settings > Installed Languages > English > Dictionaries
2. Click on '+'
3. See nothing happen, regardless of wait time.  I've waited over a minute with absolutely no response.

That stated, the app itself doesn't freeze - I can back out of that submenu at any time with no problem.  It's like it's not even trying to find anything for me to download.  This occurs on the same network I can use successfully on an emulated iPhone X test "device" for development.

**Expected behavior**
I'd expect to see _some_ acknowledgment that an online connection is being attempted or is impossible... assuming of course that I can't actually access the lexical model catalog and download our MTNT model.

**iPhone**
 - Device: iPhone SE
 - OS: 12.3.1
 - Keyman version - current alpha (https://github.com/keymanapp/keyman/commit/9dec45228a994d8bb3ad8e9de6e1e6391300f40b)
 - Target application: Keyman

**Comment by jahorton** [comment]
An emulated iPhone SE via Xcode's Simulator does not have this issue.

Note - the Simulator uses iOS 12.2 instead of 12.3.1.  Coincidence?  Not sure _why_ that would cause this, but it's the first lead I can see.

**Comment by jahorton** [comment]
The same (real) test device does not download a predictive model when downloading a new English keyboard, either.

**Comment by jahorton** [comment]
I finally managed to gain access, though after a few noteworthy things:

* This was technically something of an update for my device, but it didn't install the newer SIL EuroLatin keyboard as default.  It kept the old v1.6 EuroLatin 2.
* I was able to uninstall the old default EuroLatin 2, leaving my testing app with no permanent default keyboard.
* That "new English keyboard" earlier?  The cloud's copy of SIL EuroLatin.
* Re-entering the app, I was surprised when SIL EuroLatin was suddenly at the top of the keyboard listing.  I can still uninstall it, though.
* Attempting to install a dictionary _now_ succeeds!

We may have an issue with users updating from older versions of Keyman if this is any indication.
